src/db/models/user.js

```diff
@@ -0,0 +1,54 @@
+export default (sequelize, DataTypes) => {
+  const User = sequelize.define('User', {
+    firstName: DataTypes.STRING,
+    lastName: DataTypes.STRING,
+    userName: DataTypes.STRING,
+    email: DataTypes.STRING,
+    password: DataTypes.STRING,
+    avatarUrl: DataTypes.STRING,
+    roleId: DataTypes.UUID
+
+  }, {});
+  User.associate = (models) => {
+    User.hasMany(models.Permissions, {
+      foreignKey: 'userId',
+      onDelete: 'CASCADE'
+    });
+    User.hasMany(models.Report, {
```

Comment:
Leave new lines between all the associations
```suggestion

    User.hasMany(models.Report, {
```

Reply:
Done.